### DISCUSSION TITLE
Tidal setup?

### DISCUSSION THREAD

**Discussion by Coder84619** [RandomNinjaAtk/arr-scripts#188] [Q&A] [answered]
I set download clients to "both" and restarted the container a few times. But I parsed through all logs and I didn't see the "extra steps" needed. Where do I look for the 'extra steps'?

**Comment by RandomNinjaAtk** [comment, marked as answer]
When it triggers the tidal client, it will ask you to login. Which this can be seen by watching the logs. Unfortunately their isn’t a better way to handle it…

**Comment by Coder84619** [comment]
@RandomNinjaAtk Well I set the client to "both" and also "tidal" and I don't see anything in the logs about a login. I even downloaded the whole log and search on "tidal" and there were no login requests. I do see tidal video download entries in the log.

**Reply by githubusr80** [reply]
If it doesn’t ask you to authenticate, it doesn’t need you too. The script will exit if it’s not authenticated.

**Reply by Coder84619** [reply]
Sorry for being dense...but it's most certainly NOT downloading from Tidal. I do not see any authentication prompts.

**Reply by RandomNinjaAtk** [reply]
Then post logs showing where it processes an album…

Without logs here, nothing to be done but speculate and this is likely user error…

**Reply by Coder84619** [reply]
Can tidal-dl distinguish between "MQA" high res and non-MQA FLAC high-res (e.g. 24 bit)? I don't want any MQA crap, but I would like "high res" non-MQA tracks. "lossless" in the config file seems to top out at 16/44.

**Reply by KINGLIFER** [reply]
Did you find a solution? I asked as well and got no help on why the url is not showing up. It was clearly working previously but now no url shows up in the logs.

**Reply by thechez** [reply]
I get things similarly. I have installed the linuxserver portion, added a conf folder and the extended.conf file with tidal and the country code but when I start up my log shows this:
Linuxserver.io version: 2.10.3.4602-ls38
Build-date: 2025-04-16T02:47:06+00:00
───────────────────────────────────────
    
[custom-init] No custom files found, skipping...
[Info] Bootstrap: Starting Lidarr - /app/lidarr/bin/Lidarr - Version 2.10.3.4602 
[Info] AppFolderInfo: Data directory is being overridden to [/config] 
[Debug] Bootstrap: Console selected 
[Info] AppFolderInfo: Data directory is being overridden to [/config] 
[Info] AppFolderInfo: Data directory is being overridden to [/config] 
[Info] MigrationController: *** Migrating data source=/config/lidarr.db;cache size=-20000;datetimekind=Utc;journal mode=Wal;pooling=True;version=3;busytimeout=100 *** 
[Info] FluentMigrator.Runner.MigrationRunner: DatabaseEngineVersionCheck migrating 
[Info] FluentMigrator.Runner.MigrationRunner: PerformDBOperation  
[Info] NzbDrone.Core.Datastore.Migration.Framework.NzbDroneSQLiteProcessor: Performing DB Operation 
[Info] DatabaseEngineVersionCheck: SQLite 3.48.0 
[Info] FluentMigrator.Runner.MigrationRunner: => 0.0356514s 
[Info] FluentMigrator.Runner.MigrationRunner: DatabaseEngineVersionCheck migrated 
[Info] FluentMigrator.Runner.MigrationRunner: => 0.0376712s 
[Info] MigrationController: *** Migrating data source=/config/logs.db;cache size=-20000;datetimekind=Utc;journal mode=Wal;pooling=True;version=3;busytimeout=100 *** 
[Info] FluentMigrator.Runner.MigrationRunner: DatabaseEngineVersionCheck migrating 
[Info] FluentMigrator.Runner.MigrationRunner: PerformDBOperation  
[Info] NzbDrone.Core.Datastore.Migration.Framework.NzbDroneSQLiteProcessor: Performing DB Operation 
[Info] DatabaseEngineVersionCheck: SQLite 3.48.0 
[Info] FluentMigrator.Runner.MigrationRunner: => 0.9009935s 
[Info] FluentMigrator.Runner.MigrationRunner: DatabaseEngineVersionCheck migrated 
[Info] FluentMigrator.Runner.MigrationRunner: => 0.901936s 
[Info] Microsoft.Hosting.Lifetime: Now listening on: http://[::]:8686 
[Info] Microsoft.Hosting.Lifetime: Application started. Press Ctrl+C to shut down. 
[Info] Microsoft.Hosting.Lifetime: Hosting environment: Production 
[Info] Microsoft.Hosting.Lifetime: Content root path: /app/lidarr/bin 
[Info] ManagedHttpDispatcher: IPv4 is available: True, IPv6 will be disabled 
[ls.io-init] done.
[Info] RootFolderWatchingService: Watching directory /data/ 

I can't get it to recognize the extended script for some reason. Any ideas?
